registry/purview-registry/registry/purview_registry.py

```diff
@@ -0,0 +1,467 @@
+            ],
+            superTypes=["DataSet"],
+
+        )
+        type_feathr_sources = EntityTypeDef(
+            name=str(EntityType.Source),
+            attributeDefs=[
+
+                AtlasAttributeDef(
+                    name="path", typeName="string", cardinality=Cardinality.SINGLE),
+                AtlasAttributeDef(name="event_timestamp_column",
+                                  typeName="string", cardinality=Cardinality.SINGLE),
+                AtlasAttributeDef(name="timestamp_format",
+                                  typeName="string", cardinality=Cardinality.SINGLE),
+                AtlasAttributeDef(name="type", typeName="string",
+                                  cardinality=Cardinality.SINGLE),
+                AtlasAttributeDef(name="preprocessing", typeName="string",
+                                  cardinality=Cardinality.SINGLE),
+                AtlasAttributeDef(name="tags", typeName="map<string,string>",
+                                  cardinality=Cardinality.SINGLE),
+            ],
+            superTypes=["DataSet"],
+        )
+
+        type_feathr_anchor_features = EntityTypeDef(
+            name=str(EntityType.AnchorFeature),
+            attributeDefs=[
+                AtlasAttributeDef(name="type", typeName="string",
+                                  cardinality=Cardinality.SINGLE),
+                AtlasAttributeDef(name="key", typeName="array<map<string,string>>",
+                                  cardinality=Cardinality.SET),
+                AtlasAttributeDef(name="transformation", typeName="map<string,string>",
+                                  cardinality=Cardinality.SINGLE),
+                AtlasAttributeDef(name="tags", typeName="map<string,string>",
+                                  cardinality=Cardinality.SINGLE),
+            ],
+            superTypes=["DataSet"],
+        )
+
+        type_feathr_derived_features = EntityTypeDef(
+            name=str(EntityType.DerivedFeature),
+            attributeDefs=[
+                AtlasAttributeDef(name="type", typeName="string",
+                                  cardinality=Cardinality.SINGLE),
+                # "input_anchor_features" and "input_derived_features" are deleted, use process entity instead
+                AtlasAttributeDef(name="key", typeName="array<map<string,string>>",
+                                  cardinality=Cardinality.SET),
+                AtlasAttributeDef(name="transformation", typeName="map<string,string>",
+                                  cardinality=Cardinality.SINGLE),
+                AtlasAttributeDef(name="tags", typeName="map<string,string>",
+                                  cardinality=Cardinality.SINGLE),
+            ],
+            superTypes=["DataSet"],
+        )
+
+        type_feathr_anchors = EntityTypeDef(
+            name=str(EntityType.Anchor),
+            attributeDefs=[
+                # "source" will be removed, use process entity instead
```

Comment:
Can we not remove those fields from the definition? We can just ignore those fields, as removing those attributes will need end users completely purge all the existing purview entities.

Reply:
Ok, will put empty properties instead